EmployeeTaskSolicy/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using EmployeeTaskSolicy.Dto;
+using EmployeeTaskSolicy.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EmployeeTaskSolicy.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeeController : Controller
+    {
+        private readonly IEmployeeRepository _employeeRepo;
+
+        public EmployeeController(IEmployeeRepository employeeRepo)
+        {
+            _employeeRepo = employeeRepo;
+        }
+
+        [Route("/Error")]
+        public IActionResult HandleError() =>
+            Problem();
+
+        [HttpGet]
+        [Route("GetEmployes")]
+        public async Task<IActionResult> GetEmployes()
+        {
+            try
+            {
```

Comment:
no need for try catch, if something goes wrong it will send 500 internal error, no need to catch and send it